content/en/docs/tasks/manage-daemon/update-daemon-set.md

```diff
@@ -32,9 +32,9 @@ DaemonSet has two update strategy types:
   before.
 * RollingUpdate: With `RollingUpdate` update strategy, after you update a
   DaemonSet template, old DaemonSet pods will be killed, and new DaemonSet pods
-  will be created automatically, in a controlled fashion. 
+  will be created automatically, in a controlled fashion.
 
-## Caveat: Updating DaemonSet created from Kubernetes version 1.5 or before 
+## Caveat: Updating DaemonSet created from Kubernetes version 1.5 or before
```

Comment:
I think this caveat can be dropped because the content is about k8s 1.5, which is too old to maintain.

Reply:
@tengqm I would be happy to open a separate PR to remove this ( and will do so momentarily )  but I think it is unrelated to my added link and I'd like to keep things distinct.

Reply:
https://github.com/kubernetes/website/pull/9270